### DISCUSSION TITLE
Docs for multiple apps

### DISCUSSION THREAD

**Discussion by junevm** [requarks/wiki#5135] [Help / Questions]
Suppose I have two apps -> APP1 and APP2
I want to the wiki/documentation/API Documentation  for APP1 to be at https://docs.example.com/APP1/  and for APP2 it should be available at https://docs.example.com/APP2/
How can I achieve this behavior.

OR

How to create wikis for two different apps with same deployment.

**Comment by NGPixel** [comment]
You can have 2 sections in the same instance. No need for 2 different instances.

Simply create pages at path /app1 and /app2
